internal/generator/vector/helpers/helpers.go

```diff
@@ -63,3 +66,17 @@ func ConfigPath(name string, file string) string {
 func SecretPath(secretName string, file string) string {
 	return fmt.Sprintf("%q", filepath.Join("/var/run/ocp-collector/secrets", secretName, file))
 }
+
+func FromVectorSecretWithSecretPath(secretName, key string) string {
+	return fmt.Sprintf("SECRET[%s.%s_%s]", VectorSecretID, FormatComponentID(secretName), FormatComponentID(key))
+}
+
+func FromVectorSecretWithSecretKey(secretKey *v1.SecretKey) string {
```

Comment:
Why do we need this one and the other?  At a minim we should be able to call one from the other to reduce code duplication

Reply:
Yes, need to fix that, in previous code all `Authentication`  was `SecretKey` so was used `FromVectorSecretWithSecretKey`